release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -60,6 +60,63 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-6-rhcos"]
 === {op-system-first}
 
+[id="ocp-4-6-boot-live-iso-by-running-coreos-installer"]
+==== Boot live ISO by running `coreos-installer`
+
+You can now boot an {op-system} live ISO in a live environment by running the
+`coreos-installer`. This allows you to perform pre-install tasks like hardware
+discovery and complex network configuration. Additionally, you can create your
+own custom live ISO with embedded Ignition configs and installation
+customizations for an automated {op-system} installation.
+
+The live ISO is capable of performing installations completely offline. This
+replaces the previous installation ISO, though the same kernel arguments are
+supported.
+
+[id="ocp-4-6-refactored-coreos-installer"]
+==== Refactored `coreos-installer`
+
+The `coreos-installer` has been rewritten to support more features, some of
+which include:
+
+* Modifying the kernel arguments of the installed system.
+* Fetching Ignition configs.
+* Saving previously existing partitions.
+
+[id="ocp-4-6-ignition-spect-updated-v3"]
+==== Ignition Spec updated to v3
+
+{op-system} has switched to using Ignition spec v3 as the only supported spec
+version of Ignition. This will allow for more complex disk configuration support
+in the future.
+
+The change should be mostly transparent for those using installer-provisioned
+infrastructure. For user-provisioned infrastructure installations, you must
+adapt any custom Ignition configurations to use Ignition spec 3. The
+`openshift-install` program now generates Ignition spec 3.
+
+If you are creating Machine Configs for day 1 or day 2 operations that use
+Ignition snippets, they must be created using Ignition spec v3.
+
+[id="ocp-4-6-extensions-supported-for-rhcos-mco"]
+==== Extensions now supported for {op-system} and MCO 
+
+{op-system} and the Machine Config Operator (MCO) now support extensions to the
+default {op-system} installation. The extensions mechanism allows you to install
+additional packages on {op-system} based on your needs. The currently supported
```

Comment:
> based on your needs

This is a bit open ended and I don't want to suggest to customers that they are able to install any package they need on their RHCOS nodes.  Can we change the wording so that it is clear we only support `usbguard` and `kernel-devel`